RELEASE/data/autoscend_properties.txt

```diff
@@ -226,3 +226,4 @@
 223	auto_paranoia_counter
 224	auto_wishes
 225	auto_restoreUseBloodBond
+226	auto_towerBreak
```

Comment:
Think we don't need to add it to this file (as this file is for converting properties from pre-(when the script got renamed to auotscend)-days) - or at least I got told somethign similar when I did the same for a property I added